custodian/scripts/deploy-custodian.sh

```diff
@@ -6,6 +6,7 @@ COST_CENTRE=$(aws ssm get-parameter --name /mgmt/cost_centre | jq '.Parameter.Va
 SLACK_WEBHOOK=$(aws ssm get-parameter --name /mgmt/slack/webhook | jq '.Parameter.Value' | tr -d '"')
 HOSTED_ZONE=$(aws route53 list-hosted-zones --max-items 1 | jq '.HostedZones[0].Name' | tr -d '"' | sed 's/.$//')
 CUSTODIAN_REGION="eu-west-2"
```

Comment:
Should this be changes to: CUSTODIAN_REGION_1?

Reply:
thanks, updated as suggested